scio-test/src/it/scala/com/spotify/ScioBatchBenchmark.scala

```diff
@@ -366,7 +366,9 @@ abstract class Benchmark(val extraConfs: Map[String, Array[String]] = null) {
 
   def run(projectId: String,
           prefix: String,
-          args: Array[String]): Iterable[Future[BenchmarkResult]] = {
+          args: Array[String],
+          scioVersion: String,
```

Comment:
`scioVersion`/`beamVersion` probably don't even have to be passed in as function args, they can be computed in `run()` itself like `username` is. wdyt?